docs/release_notes/DRAFT_Release_notes_0.0.4.adoc

```diff
@@ -18,6 +18,7 @@
 - Added SCA approaches and IDP url columns on the registry UI
 - Added swagger documentation for PSD2/Facade API, OAuth API and ASPSP Registry API
 - Added PERSIST endpoint to `aspsp-registry` for replacing an original CSV of Aspsps with a new updated one. It is available only via `dev` profile
+- Added Pagination feature to UI
```

Comment:
should point that feature was implemented for aspsp registry

Reply:
got it